shared-helpers/__tests__/pdfs.test.ts

```diff
@@ -6,14 +6,14 @@ afterEach(cleanup)
 
 describe("pdfs helpers", () => {
   const OLD_ENV = process.env
-  process.env.CLOUDINARY_CLOUD_NAME = "exygy"
+  process.env.cloudinaryCloudName = "exygy"
 
   afterAll(() => {
     process.env = OLD_ENV
   })
 
   it("should format cloudinary url", () => {
-    expect(cloudinaryPdfFromId("1234")).toBe(
+    expect(cloudinaryPdfFromId("1234", "exygy")).toBe(
```

Comment:
supernit suggestion: to make sure this test works I would use a different string than "exygy" -- since that is just the value of process.env.cloudinaryCloudName